### PR TITLE
fs-1738-global translations

### PR DESCRIPTION
**Pull request by donshellywork**
rel ticket: https://digital.dclg.gov.uk/jira/browse/FS-1738?workflowName=Funding+Services+%28FS%29&stepId=1

Added Welsh translation to the global elements (Header and Footer).
Added the translation files and file structure.
Added python logic to create_app.py that initializes the language support.
Added python logic that interprets the language via manual URL changes e.g. localhost:5000/language/cy
Correct Welsh translations are still needed, they have been requested via the excel spreadsheet

### CODE REVIEW

**Review by gio-karunakaran — APPROVED** [review]
Great stuff!

**Review by gidsg — CHANGES_REQUESTED** [review]
We should not be using translations from Google Translate, can we park this until we get human translations back?

File: translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+
+#: frontend/templates/header.html:39
+msgid "feedback"
+msgstr "adborth"
+
+#: frontend/templates/header.html:39
+msgid "will help us to improve it."
+msgstr "yn ein helpu i'w wella."
+
+#: venv/Lib/site-packages/click/_termui_impl.py:496
```
Comment (gidsg) [inline translations/cy/LC_MESSAGES/messages.po:66]: We should not be pulling through localisation strings from libraries.
Please can we add a `babel.cfg` to exclude these, as in https://github.com/communitiesuk/funding-service-design-frontend/pull/179

Comment (donshellywork) [reply translations/cy/LC_MESSAGES/messages.po:66]: will do

**Review by gidsg — CHANGES_REQUESTED** [review]
Don't think this is actually working when checked out, so don't think it should be merged until fixed.

File: frontend/templates/header.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <header class="govuk-header " role="banner" data-module="govuk-header">
     <div class="govuk-header__container govuk-width-container">
         <div class="govuk-header__logo">
-            <a class="govuk-header__link govuk-header__link--homepage">
+            <a class="govuk-header__link govuk-header__link--homepage" style="pointer-events: none;">
```
Comment (gidsg) [inline frontend/templates/header.html:6]: Any idea where this style change has come from? Doesn't seem related to translations? Can we remove it?

Comment (gio-karunakaran) [reply frontend/templates/header.html]: not sure where that came from 🤔 but have removed this now 👍🏾

**Review by gidsg — APPROVED** [review]
Nice one 👍 🏴󠁧󠁢󠁷󠁬󠁳󠁿
